### PR TITLE
[WAYANG-1][WAYANG-2] Parent artifact and modification code structure

### PR DESCRIPTION
**Pull request by berttty**
correction on the code structure and also generation of a parent module

### CODE REVIEW

File: wayang-api/pom.xml [hunk 1]
```diff
@@ -15,94 +15,109 @@
...
-            </plugin>
-        </plugins>
-    </build>
 
     <dependencies>
         <dependency>
             <groupId>org.apache.wayang</groupId>
             <artifactId>wayang-core</artifactId>
-            <version>1.0.0-SNAPSHOT</version>
+            <version>${wayang.core.version}</version>
```
Comment (chrisdutz) [inline wayang-api/pom.xml]: Using expressions like this "<version>${wayang.core.version}</version>" tend to cause more problems than they solve ... I would strongly reccomend to stick to using plain-text versions. The Release-plugin updates them for you anyway.

**Review by chrisdutz — CHANGES_REQUESTED** [review]

File: wayang-api/pom.xml (hunk 1 above)
Comment (berttty) [reply wayang-api/pom.xml]: Ok, I will change it

**Comment by berttty** [comment]
I did the changes on the variables

**Comment by ro-pardo** [comment]
Test are running locally

**Pull request merged**